### PR TITLE
feat: bridge tx overview

### PR DESCRIPTION
**Pull request by matt-user**
closes FRO-114

### CODE REVIEW

**Review by luizstacio — APPROVED** [review]

File: packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { cssObj } from '@fuel-ui/css';
+import { Box } from '@fuel-ui/react';
+import { bn } from 'fuels';
+
+import ethImage from '../../../../../public/eth-logo.svg';
```
Comment (luizstacio) [inline packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx:5]: Can we use `~public` to `avoid ../.,......`

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx (hunk 1 above)
Comment (matt-user) [reply packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx:5]: I'll try your suggestion.  IIRC we need to explicitly import images like this for storybook.

Comment (matt-user) [reply packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx:5]: your suggestion did not work.  I tried looking in other repos and the storybook docs for an example of how to handle this in stories, but I haven't been able to find any examples.

Comment (luizstacio) [reply packages/app/src/systems/Bridge/components/BridgeTxOverview/BridgeTxOverview.stories.tsx:5]: You probably need to configure the alias from tsconfig on the storybook a example can be found on the fuels-wallet repo.

**Review by luizstacio — APPROVED** [review]

**Pull request merged**